### PR TITLE
Update for homework Variables

### PR DESCRIPTION
**Pull request by polinaK4**

### CODE REVIEW

File: PolinaProject/Program.cs [hunk 1]
```diff
@@ -82,7 +83,11 @@
 
 Console.Write("Task 8. Write any number: ");
 double variableA = double.Parse(Console.ReadLine());
-if ((variableA < 50 && variableA != 37 && variableA > 32) || variableA == 0 || variableA == 15)
+if (variableA < 50 && variableA != 37 && variableA > 32)
+{
+    Console.WriteLine($"Working");
+}
+else if (variableA == 0 || variableA == 15)
```
Comment (pickslayer17) [inline PolinaProject/Program.cs:90]: here it's better to revert this change to avoid using code repeating. We don't want to repeat the same code parts as Console.WriteLine($"Working") for example

File: PolinaProject/Program.cs [hunk 2]
```diff
@@ -82,7 +83,11 @@
 
 Console.Write("Task 8. Write any number: ");
 double variableA = double.Parse(Console.ReadLine());
-if ((variableA < 50 && variableA != 37 && variableA > 32) || variableA == 0 || variableA == 15)
```
[the remark this answers is not in the document]
Comment (polinaK4) [reply PolinaProject/Program.cs:85]: "but not less than or equal to 32" - ok it seems like I'm confused a little.. is it really "variableA !< 32 || variableA != 32" ?

[the remark this answers is not in the document]
Comment (pickslayer17) [reply PolinaProject/Program.cs:85]: nu well, its not so important. You can use !(variable <= 32) here
But actually I couldn't find any case where this difference could affect something
